fake_secrets.yaml

```diff
@@ -57,3 +57,65 @@
 - name: papertrail_logging_secret
   path: /etc/taskcluster/secrets/papertrail_secret
   value: "f2f3475398392f2783b8fd11c7a9fe1914346ed9"
+# cert for taskcluster-worker.net
```

Comment:
This doesn't actually need to be for `*.taskcluster-worker.net` -- it needs to be for the domain specified by `statelessHostname.domain`.  Some comments about that might help the next person :)